test/functional/page_objects/login_page.ts

```diff
@@ -48,10 +48,7 @@ export function LoginPageProvider({ getService }: FtrProviderContext) {
 
   class LoginPage {
     async login(user: string, pwd: string) {
-      if (
-        process.env.VM === 'ubuntu18_deb_oidc' ||
-        process.env.VM === 'ubuntu16_deb_desktop_saml'
-      ) {
+      if (process.env.VM === 'ubuntu18_deb_oidc' || process.env.VM === 'ubuntu18_deb_saml') {
```

Comment:
should we just check if the process.env.VM name includes `oidc` or `saml` ?  That way we could add/remove/change VMs without having to fix this?